.github/workflows/combine.yml

```
@@ -0,0 +1,55 @@
+name: combine
+
+on:
+  pull_request:
+  # Allows you to run this workflow manually
+  workflow_dispatch:
+
+jobs:
+  combine:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ github.head_ref }}
```

head_ref is only available on pull_requests, will need `github.sha` as a fallback for the manual dispatch
```suggestion
          ref: ${{ github.head_ref || github.sha }}
```